src/content/Angular-CSharp/ClientApp/angular.json

```diff
@@ -40,8 +40,8 @@
               "budgets": [
                 {
                   "type": "initial",
-                  "maximumWarning": "500kb",
-                  "maximumError": "1mb"
+                  "maximumWarning": "1mb",
```

Comment:
This is CLI generated, correct? And not us manually updating these?

Reply:
This was manually updated. This fixes a publish warning that's been around for a while that says the bundle size exceeds this specified maximum. It's pretty difficult for us to keep the bundle size under 500kb because `@angular` and `oidc-client` alone occupy 482.94kb, which gives us almost very little head room to shave things down.